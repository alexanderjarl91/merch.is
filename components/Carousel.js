import React, { useState } from "react";
import styles from "../styles/Carousel.module.css";
import Image from "next/image";
import Link from "next/link";

export default function Carousel({ toggleSignUp }) {
  return (
    <>
      <div>
        <div className={styles.frontpageGrid}>
          {/* Image grid for desktop ladning frontpage */}
          <div className={styles.frontpage_img}>
            <Image src="/frontpage_img.png" width={375} height={460}></Image>
          </div>
          <div className={styles.headlineGrid}>
            <p className={styles.front_headline}>SELDU ÞÍNAR VÖRUR</p>
          </div>

          <div className={styles.descriptionGrid}>
            <p className={styles.front_description}>
              Byrjaðu að selja vörurnar þínar strax í dag!
            </p>
          </div>

          <div className={styles.nyskraGrid}>
            <button
              className={styles.nyskraBtn}
              onClick={() => {
                toggleSignUp();
              }}
            >
              Nýskrá
            </button>
            <p className="login_paragraph">
              eða{" "}
              <strong>
                <Link href="/">
                  <a>skráðu þig</a>
                </Link>
              </strong>{" "}
              {""}
              inn með þínum aðgangi
            </p>
          </div>
        </div>
      </div>
      {/* </div> */}
    </>
  );
}
