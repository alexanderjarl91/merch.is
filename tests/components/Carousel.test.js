import { shallow } from "enzyme";
import React from "react";
import Carousel from "../../components/Carousel";

it("expect Carousel component to match Snapshot", () => {
  expect(shallow(<Carousel />)).toMatchSnapshot();
});

it("expect to render component Carousel", () => {
  expect(shallow(<Carousel />).length).toEqual(1);
});
