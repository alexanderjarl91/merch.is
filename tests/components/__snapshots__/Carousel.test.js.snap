// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect Carousel component to match Snapshot 1`] = `ShallowWrapper {}`;
